setup_builder.py

```diff
@@ -646,7 +671,7 @@ def _assert_github_full_repo(arg: str) -> str:
     args = parser.parse_args()
 
     main(
-        args.setup_cfg_file,
+        Path(args.setup_cfg_file),
```

Comment:
Consider doing the Path conversion on line 632, in the type checking.  Not required.